bridge/contracts/nftbridge/ISideNFTToken.sol

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: MIT
 
 pragma solidity ^0.7.0;
+pragma abicoder v2;
```

Comment:
maybe I'd add a comment in some of the README.md files (or in every contract that uses this) explaining [what it's for](https://ethereum.stackexchange.com/a/64566)

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4